lib/rules/no-useless-flag.ts

```diff
@@ -758,6 +772,32 @@ function createRegExpReferenceExtractVisitor(
     )
 }
 
+/**
+ * Parse option
+ */
+function parseOption(
+    userOption: {
+        ignore?: ("i" | "m" | "s" | "g" | "y")[]
+        strictTypes?: boolean
+    } | void,
```

Comment:
Kinda the same but different.

```suggestion
    } | undefined,
```